packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -244,12 +244,15 @@ export function parseWebViewType(webViewString?: string): devSettings.WebViewTyp
     case "internet explorer":
     case "internetexplorer":
       return devSettings.WebViewType.IE;
-    case "edge":
+    case "edgelegacy":
+    case "edge-legacy":
     case "spartan":
```

Comment:
Do we really want to accept multiple names for a single webview type?

Reply:
It's fine to have the flexibility. We were already doing this.

Reply:
It's what we've been doing before, and all of these names refer to the same specific webview. I don't see the issue with accepting multiple names, but if there's something that it specifically affects, we can change it.

Reply:
All these names get converted to the same WebViewType enum, so it gets handled the same way.